llvm/include/llvm/TableGen/Record.h

```diff
@@ -781,10 +782,12 @@ class ListInit final : public TypedInit,
 
   void Profile(FoldingSetNodeID &ID) const;
 
-  const Init *getElement(unsigned i) const {
-    assert(i < NumValues && "List element index out of range!");
-    return getTrailingObjects<const Init *>()[i];
+  ArrayRef<const Init *> getValues() const {
+    return ArrayRef(getTrailingObjects<const Init *>(), NumValues);
   }
+
+  const Init *getElement(unsigned i) const { return getValues()[i]; }
```

Comment:
Non-standard naming (it's cut & paste, but as long as we're cleaning up...)

Reply:
Done. I assume you meant the name `i`.